spec/system/search_game_rooms_spec.rb

```diff
@@ -3,7 +3,9 @@
 RSpec.describe 'SearchGameRooms', type: :system do
   include_context 'when user is logged in'
   let(:track) { create(:track, artist: 'dua lipa') }
```

Comment:
### Note 2
Since artist is the same with our test data  (Refer to `test/factories/tracks.rb`), you can remove the artist 😃 

```suggestion
  let(:track) { create(:track) }
```